### PR TITLE
Claims API Status

### PR DESCRIPTION
**Pull request by ATeal**
## Description of change
- Give the claims API it's own serializer
- Remove phase and add status
- Map the phase to status in the claims api model

resolves https://github.com/department-of-veterans-affairs/vets-contrib/issues/882

## Testing done
- rspec

## Acceptance Criteria (Definition of Done)
- The API returns a status that is similar to the one displayed on the va.gov website

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)

### CODE REVIEW

File: modules/claims_api/app/models/claims_api/evss_claim.rb [hunk 1]
```diff
@@ -5,6 +5,19 @@ class EVSSClaim
     include Virtus.model
     include ActiveModel::Serialization
 
+    EVIDENCE_GATHERING = 'Gathering of evidence'
```
Comment (dhall) [inline modules/claims_api/app/models/claims_api/evss_claim.rb]: What would you think about providing a comment and link that points back to the front end code as the source for this mapping?

Comment (ATeal) [reply modules/claims_api/app/models/claims_api/evss_claim.rb]: I _think_ that makes sense... My question would be where do we want the source of truth to be, if we're always mirroring what the front end does I'm fine with that. If the intent is for us to become the source of truth, I would think they would reference us

Comment (ATeal) [reply modules/claims_api/app/models/claims_api/evss_claim.rb]: I agree! I'll add a block comment with some description

Comment (ATeal) [reply modules/claims_api/app/models/claims_api/evss_claim.rb]: https://github.com/department-of-veterans-affairs/vets-api/pull/2701/commits/6a201d61ccbf1bec8a982f3855d9e00f166ca008

@dhall I think that encapsulates what we discussed

**Pull request merged**